lib/glific/templates/interactive_templates.ex

```diff
@@ -621,6 +621,9 @@ defmodule Glific.Templates.InteractiveTemplates do
   defp do_items_translated(content, items_translations) do
```

Comment:
There are refactoring oppurtunities for this function, basically we should go through the options and then into the items at once rather than with multiple Enums. We can discuss this if we want to do that in this PR or another.